docs/integrations/bitbucket.rst

```diff
@@ -0,0 +1,28 @@
+Bitbucket
+=========
+
+Sentry integrates with both Bitbucket Issues and Pipelines.
+
+
+Bitbucket Issue Creation from Sentry
```

Comment:
I prefer to make titles in docs actions since they indicate things you can do. So this one I'd write as:

*Set up Bitbucket Issue creation in Sentry*